crates/extensions/c8y_mapper_ext/src/converter.rs

```diff
@@ -187,6 +181,7 @@ pub struct CumulocityConverter {
     pub auth_proxy: ProxyUrlGenerator,
     pub downloader_sender: LoggingSender<IdDownloadRequest>,
     pub pending_operations: HashMap<CmdId, SmartRestOperationVariant>,
+    pub inventory_model: Value,
```

Comment:
Sure. This `inventory_model` is a live copy of the aggregated inventory that the mapper builds by aggregating the twin data from all the `twin/xyz` endpoint. We're currently using it to avoid mapping duplicate fragment update messages. In future, this will be persisted on the disk as well, and will even be synced with the cloud on startup.